models/articles-model.js

```diff
@@ -6,3 +6,13 @@ exports.articleById = (article_id) => {
         return rows[0];
     });
 };
+
+exports.allArticles = () => {
+    return db
+        .query(
+            "SELECT articles.article_id, articles.title, articles.topic, articles.author, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.body) AS comment_count FROM articles LEFT JOIN comments USING (article_id) GROUP BY articles.article_id ORDER BY articles.created_at DESC"
```

Comment:
You only need to specify the table the column comes from if the column name belongs in both tables (eg. you don't need to put "articles." on most of these.